src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -913,9 +857,64 @@ bool AsyncRPCOperation_sendmany::find_utxos(bool fAcceptCoinbase=false) {
         return i.Value() < j.Value();
     });
 
+    // Load transparent inputs
+    load_inputs(txValues);
+
     return t_inputs_.size() > 0;
 }
 
+bool AsyncRPCOperation_sendmany::load_inputs(TxValues& txValues) {
+    // If from address is a taddr, select UTXOs to spend
+    CAmount selectedUTXOAmount = 0;
```

Comment:
Do we need `txValues.selectedUTXOCoinbase = false;` here?

Reply:
you mean to set `selectedUTXOCoinbase` to false at the beginning of the method?.

The `TxValues` object is created right before calling `find_utxos` (which is called only once per operation process), initializing `selectedUTXOCoinbase` to false by default (check the struct), then the flag is only changed here if a coinbase utxo gets selected.
Plus, `txValues` is not cached nor reused for any other operation process, dies here.